src/app/components/Header/Header.tsx

```diff
@@ -28,8 +47,9 @@ const Header: FC = () => {
         console.log("RawwwwwPrismic response for blogpost:", response);
```

Comment:
can remove this console.log as we can see prismic response in network tab of chrome dev tools, no?